core-plugins/src/main/java/co/cask/hydrator/plugin/batch/action/SSHAction.java

```diff
@@ -84,7 +84,9 @@ public void run(final ActionContext context) throws Exception {
       String err = CharStreams.toString(errBuffer);
 
       LOG.debug("Output: {}", out);
-      LOG.error("Errors: {}", err);
+      if (err.length() > 0) {
+        LOG.error("Errors: {}", err);
```

Comment:
its also more accurate to call this stderr. Lots of times people aren't very judicious about what goes to stdout vs stderr